test/e2e/harness.go

```diff
@@ -54,10 +54,10 @@ func getConfig() E2EConfig {
 	stressTestFailureRate := int64(2)
 	stressTestTargetTPS := float64(700)
 
-	if !bootstrap {
+	if !isBootstrap {
```

Comment:
I disagree with this rename. If `bootstrap` is problematic, use `shouldBootstrap`

Reply:
changed.